fft_folder/inv_fft.py

```diff
@@ -0,0 +1,104 @@
+"""
+inv_fft.py
+inverse fast fourie transform
+"""
```

Comment:
this should be renamed to `preparation.py`cans should container at least three functions:
1. fft
2. inverse fft
3. calculate frequencies

Reply:
so the task was to make my own fft function? and not use the numpy functions?